theme/islandora-videojs.tpl.php

```diff
@@ -1,15 +1,23 @@
-<video id="islandora_videojs" class="video-js vjs-default-skin" controls
- preload="auto" width="640" height="264" 
+<video id="islandora_videojs" <?php if (empty($responsive)): ?>width="640" height="360"<?php endif; ?>
+  <?php if (empty($center_play_button)): ?>class="video-js vjs-default-skin"
```

Comment:
@pautri Same as comment as above about using the boolean nature of the variable, but with the `$center_play_button`. ie. Change `if (empty($center_play_button))` to a `if (!$center_play_button)` to make use of the new boolean nature of the variable.